dependencies.txt

```diff
@@ -1,9 +1,9 @@
 django
-django-reversion <= 1.10.2
-djangorestframework >= 3.5.3
-django-debug-toolbar >= 1.6
+django-reversion == 2.0.10
```

Comment:
Pinning to a specific version seems very strict. If we want to go with latest django we might require `django >= 1.8` and `reversion-reversion >= 2.0.0`.

see reversions [compatibility matrix](https://django-reversion.readthedocs.io/en/stable/django-versions.html)

Reply:
true